samples/bluetooth/central_path_loss_monitoring/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  description: Central device of a Bluetooth Low Energy proximity sensor using path loss monitoring
+  name: Bluetooth LE central proximity sensor
```

Comment:
I don't think we want to call it "proximity sensor" as this may be over-promising what path loss monitoring delivers. Maybe we just stick with path loss monitoring sample.